src/modules/addScore.js

```diff
@@ -0,0 +1,17 @@
+// import { addLocalStorage } from './localStorage.js';
+import Displaying from './displayScore.js';
+import { add } from '../data/api.js';
+
+const user = document.querySelector('section .add-score form input#name');
+const score = document.querySelector('section .add-score form input#score');
+const displaying = new Displaying();
+function addNewScore() {
+  const obj = {
+    user: user.value,
+    score: score.value,
+  };
+  add(obj);
+  displaying.displayScore(obj);
+}
```

Comment:
- [ ] Kindly change all the functions in your code to the arrow function instead of the keyword function. This is part of the project requirements. Take a look please :point_down: 

![project-req](https://user-images.githubusercontent.com/26597432/202419490-f27aeaad-d457-4276-8a63-cdc6cc08c805.png)